lbry/wallet/network.py

```diff
@@ -432,7 +436,17 @@ async def new_resolve(self, server, urls):
             return result['result']
 
     async def new_claim_search(self, server, **kwargs):
+        if os.environ.get("GO_HUB") and os.environ.get("GO_HUB") == "true":
```

Comment:
This can probably be simplified, there are no other use cases of `new_claim_search` other than the new go hub. So if this function is being called is for the go hub.